packages/dm-core-plugins/src/header/components/UserInfoDialog.tsx

```diff
@@ -147,6 +147,25 @@ export const UserInfoDialog = (props: UserInfoDialogProps) => {
               >
                 Create API-Key
               </Button>
+              <Button
```

Comment:
Is this supposed to be available for all user-roles? Feel like it should be somewhat restricted

Reply:
Agreed. Updated HeaderConfig.json to now include "adminRole". If the user has that role, the "admin menu" is shown in the top bar :+1: